packages/framework-types/src/config.ts

```diff
@@ -104,6 +105,12 @@ export class BoosterConfig {
     return value
   }
 
+  public addConfiguredEnvironment(environmentName: string): void {
+    if (!this.configuredEnvironments.includes(environmentName)) {
+      this.configuredEnvironments.push(environmentName)
+    }
```

Comment:
You can use a `Set<string>` instead of an array for the `configuredEnvironments` property and then you can omit the `if` 😃 (as Sets can't contain duplicates)
```typescript
this.configuredEnvironments.add(environmentName)
```

Reply:
Done! Thank you for pointing this out.